ballerina/build.gradle

```diff
@@ -93,11 +93,61 @@ clean {
     delete 'build'
 }
 
+task startLdapServer() {
+    doLast {
+        if (!Os.isFamily(Os.FAMILY_WINDOWS)) {

```

Comment:
Aren't there any docker images that are compatible with Windows?

Reply:
The current one is compatible with Windows. Fixed in 4cd6dfe

Reply:
Revert the fix as Windows runner does not support running docker images